qiita_pet/handlers/auth_handlers.py

```diff
@@ -66,7 +66,18 @@ def post(self):
                 self.redirect(u"%s/?level=danger&message=%s"
                               % (qiita_config.portal_dir, url_escape(msg)))
                 return
-            self.redirect(u"%s/" % qiita_config.portal_dir)
+
+            msg = ("<H1><p>User Successfully Created</H1>Your Qiita account "
```

Comment:
- [ ] Can you make sure to use all lower case or all upper case (I prefer lower) for the html? You have H1 and p
- [ ] Can you make sure that you open and close all <p>? See: `<H1><p>User Successfully Created</H1>`
- [ ] I think H1 is too large, what about H3? This is up to you.

Reply:
Thanks! Tried h2 as well, felt it was too in the middle, so I switched to h3.